web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Setup/page/SetupForm/index.tsx

```diff
@@ -501,6 +511,23 @@ export const SetupForm = (props: LinksFormProps) => {
             }
             placeholder="0x12312321..., 0x12312312..."
             register={register("whitelisted_addresses")}
+            onChange={(e) => {
```

Comment:
@andy-t-wang Looks like this function is used multiple times, I think we can move it under single function above.

Reply:
ah yeah